tests/installation/first_boot.pm

```diff
@@ -20,6 +20,12 @@ sub run() {
         return;
     }
 
+    # return at the DM and log in later into awesome
+    if (check_var('DESKTOP', 'awesome')) {
```

Comment:
this is wrong too. if awesome indicates NOAUTOLOGIN, then either the template or main.pm should set it - instead of duplicating code (I refrain from sharing a link why duplicating code is bad)